pkg/driftctl_test.go

```diff
@@ -64,7 +64,9 @@ func runTest(t *testing.T, cases TestCases) {
 				c.mocks(resourceFactory)
 			}
 
-			driftctl := pkg.NewDriftCTL(remoteSupplier, stateSupplier, filter, testAlerter, resourceFactory)
+			driftctl := pkg.NewDriftCTL(remoteSupplier, stateSupplier, testAlerter, resourceFactory, &pkg.ScanOptions{
+				Filter: filter,
```

Comment:
@sundowndev Just a reminder to add a test of strict mode there (do not do this in this PR)